ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -238,8 +251,15 @@ def create_storage_cluster(self):
         Review and Create storage cluster
```

Comment:
Log relevant message for 4.9 as we now have StorageSystem

Reply:
done